p2p/pex/pex_reactor.go

```diff
@@ -150,6 +150,8 @@ func (r *PEXReactor) OnStart() error {
 
 	r.seedAddrs = seedAddrs
 
+	r.Logger.Info("Peer Info", "numPeers", r.Switch.Peers().Size())
+
```

Comment:
Won't this oly get printed in the start only?
We need it at intervals through out the running node

Reply:
I see, yes. Only this iteration or the other one as well?